site/_wiki/Documentation/ConfigurationGuide.md

```diff
@@ -0,0 +1,418 @@
+  0.7 (`master`) differs slightly. Make sure that you pick a configuration of
+  the same format as the version of OpenTabletDriver you're using.
+
+[Configuration skeleton]: https://github.com/OpenTabletDriver/OpenTabletDriver/issues/3249
+
+## Setting specifications according to official manufacturer specifications
+
+Setting up some of the specifications now can simplify touching up later values
+if you know what they are. You can also add them as you discover the feature;
+we will ensure the specifications are set correctly at the end of the guide.
+
+Some (almost) always correct values include:
+- Pen and tablet (aux) button counts
+- Tool support (such as tablet mouse)
+- Touch support
+- Pen pressure range
+
+Digitizer dimensions (for both pen and touch digitizer) will be determined as a
+later part of the guide, and is not always correct.
+
+Note that wheels (rings) are not yet supported, and should not be directly defined
+(even as a button). See [OpenTabletDriver#1367] for more information
+
+[OpenTabletDriver#1367]: https://github.com/OpenTabletDriver/OpenTabletDriver/issues/1367
+
+## Configuring the digitizer identifier
+
+Configuring a correct digitizer identifier is the bread and butter of tablet
+detection. This section covers how to set all the identifier fields
+appropriately.
+
+A single tablet model generally only needs a single identifier. Still, firmware
+variants for an otherwise identical tablet can cause tablets to need more than
+1 identifier to correctly identify variants.
+
+Therefore, it is important to make your identifier as relaxed as possible while
+still being strict enough to not cause identifier conflicts with other
+configurations.
+
+Detection, in broad terms, checks in the following order:
+
+1. Match VendorID and ProductID
+2. If not null, match InputReportLength and OutputReportLength
+  - This helps select a specific interface of the tablet
+3. If not null, match DeviceStrings
+  - This helps differentiate between firmwares
+
+There is more to detection than this (such as attributes), but 99.9% of use
+cases can be covered with the above.
+
+### ID's and report lengths
+
+At the very minimum, your configuration should have a non-null `VendorID` and `ProductID`.
+
+If your tablet has multiple interfaces, it is very likely only 1 of them
+contains the tablet data stream. This is where `InputReportLength` and
+`OutputReportLength` comes in.
+
+Even if your tablet only has a single interface, it is often a good idea to
+specify at least one of the lengths as non-zero, and only define them as `null`
+if you know what you're doing
+
+> If you specify report lengths as null, it can be hard to fix any future
+  conflicts that may arise without knowing what values your specific tablet
+  has. If you do decide to specify them as null, please make sure to include a
+  OpenTabletDriver diagnostics with the pull request, or include the lengths in
+  the commit description.
```

Comment:
Diagnostics should always be included as specified within the CONTRIBUTING.md, it can be extremely useful especially with the wider options with the information in 0.6.4.0 onwards. 

Not sure anything really needs to be changed here but I thought I might mention it while I'm reading it.